build/split_man.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+

```

Comment:
Can you add a comment saying `split_man.py ... A script that generates language-specific manpages by substituting a selected set of words in the input manpage` or something like that? Just to be as neat as rst2man.py and rst2text.py.

Reply:
Sounds good.